src/main/java/org/apache/sysds/runtime/instructions/cp/ReorgCPInstruction.java

```diff
@@ -83,6 +85,31 @@ private ReorgCPInstruction(Operator op, CPOperand in, CPOperand out, CPOperand c
 		_col = col;
 		_desc = desc;
 		_ixret = ixret;
+		_shift = new CPOperand();
+	}
+
+	/**
+	 * for opcode roll
+	 *
+	 * @param op
+	 *            operator
+	 * @param in
+	 *            cp input operand
+	 * @param shift
+	 *            ?
+	 * @param out
+	 *            cp output operand
+	 * @param opcode
+	 *            the opcode
+	 * @param istr
+	 *            ?
+	 */
+	private ReorgCPInstruction(Operator op, CPOperand in, CPOperand out, CPOperand shift, String opcode, String istr) {
+		super(CPType.Reorg, op, in, out, opcode, istr);
+		_col = new CPOperand();
+		_desc = new CPOperand();
+		_ixret = new CPOperand();
+		_shift = shift;
```

Comment:
we could consider reusing some of the other ones, 
however, If you want to keep the structure like this, then assign the other CPOperands to null if you do not use them instead of assigning new object that we will not use.

Reply:
Modified the code to initialize the unused variable to null.